docs/tutorials/core-api-tutorial.rst

```diff
@@ -0,0 +1,98 @@
+.. _core_api_tutorial_part_1:
+
+##################################################################################
+Quickstart: Integrate an Existing Training Script with the Determined Environment
+##################################################################################
+
+.. meta::
+  :description: Learn how to take an existing training script and integrate it with Determined.
+  :keywords: Core API,model developer
+
+In this tutorial, we’ll show you how to take an existing training script and integrate it with the Determined environment. We'll run our experiment on a local training environment requiring only a single CPU or GPU.
+This tutorial is recommended as an introduction for model developers who are new to Determined AI.
+
+**Objective**
+
+Our goal is to increment an integer. This will give you an idea of how you can modify your own training script to integrate it with the Core API.
+
+**Prerequisites**
+
+* `System Requirements <https://docs.determined.ai/latest/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.html#system-requirements>`_
+* `Docker <https://docs.determined.ai/latest/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.html#install-docker>`_
+
+*************************
+Get the Quickstart Files
+*************************
+
+To get started, add the quickstart files to your computer. 
```

Comment:
should be a link to the files on github
https://github.com/determined-ai/determined/tree/master/examples/tutorials/core_api